Sources/SegmentOptimizelyFullStack/OptimizelyFullStack.swift

```diff
@@ -137,6 +137,21 @@ public class OptimizelyFullStack: DestinationPlugin {
         let trackKnownUsers = _optimizelySettings?.trackKnownUsers
         var userId = event.userId
         
+        var optimizelyEvent = event
+
+        if var properties = event.properties?.dictionaryValue {
+            if let revenue = properties["revenue"] {
+                if let actualRevenue = revenue as? Int {
+                    let optimizelyRevenue = actualRevenue * 100
+                    properties["revenue"] = optimizelyRevenue
+                } else if let actualRevenue = revenue as? Double {
+                    let optimizelyRevenue = Int(actualRevenue) * 100
```

Comment:
Thanks for getting this in, but this calculation is incorrect. Should be
`let optimizelyRevenue = Int(actualRevenue * 100)`

Reply:
@skydivedan oops, that is embarrassing. Thanks for the catch. fixed in `1.2.1`